aldryn_redirects/admin.py

```diff
@@ -17,13 +17,25 @@
 from .models import Redirect, StaticRedirect, StaticRedirectInboundRouteQueryParam
 
 
+def delete_selected(modeladmin, request, queryset):
```

Comment:
I think is better for this to be a method instead of standalone function.
Also where is the limiting? I see you're doing `queryset.all().delete()` shouldn't it be `queryset[:limit].delete()`?

Reply:
as for method: agreed, I'll change that.
as for limiting: do we need it? based upon our conversations I thought there would be no need to do that. What do you think?